Kentor.AuthServices.Tests/WebSSO/SignInCommandTests.cs

```diff
@@ -134,5 +137,27 @@ public void SignInCommand_Run_ReturnsRedirectToDiscoveryService()
 
             result.Location.Should().Be(expectedLocation);
         }
+
+        [TestMethod]
+        public void SignInCommand_Run_MapsAssertionConsumerServiceURL()
+        {
+            KentorAuthServicesSection.Current.AllowChange(true);
+            KentorAuthServicesSection.Current.AssertionConsumerServiceUrl = new Uri("https://testurlForAcs.com/");
+
+            try
+            {
+                Assert.IsNotNull(KentorAuthServicesSection.Current.AssertionConsumerServiceUrl);
```

Comment:
Please use FluentAssertion instead of Assert. to follow the convention of the rest of the tests.